features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/LifeCycle/LifeCycleUpdate.jsx

```diff
@@ -194,7 +194,7 @@ class LifeCycleUpdate extends Component {
                     displayName,
                     disabled:
                         api.endpointConfig == null
-                        || api.policies.length === 0
+                        || (api.policies.length === 0 && !api.securityScheme.includes('mutualssl_mandatory'))
```

Comment:
Shall we take this value from `apiSecurityConstants`

Reply:
Publish button should be disabled if the certificate is not provided as well if mutual SSL is enabled. We need to check for this condition.